script.js

```diff
@@ -0,0 +1,14 @@
+function hideTableContents(className) {
+  const allCellsInClass = document.getElementsByClassName(className);
+
+  for (let cellIterator = 0; cellIterator < allCellsInClass.length; cellIterator++) {
```

Comment:
Do this with a for of loop.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of?retiredLocale=de